hexa/plugins/connector_s3/templates/connector_s3/object_detail.html

```diff
@@ -1,24 +1,38 @@
 {% extends "layouts/page.html" %}
-{% load embed %}
+{% load embed static %}
 {% block page_title %}{{ object.display_name }}{% endblock %}
 
 {% block page_content %}
-    {{ object_card }}
-    {% include "comments/components/page_section_comments.html" with object=object %}
-    {% if object.type == 'directory' %}
-        {% embed "ui/section/section.html" with title=_("Objects") %}
-            {% slot extra_wrapper_attrs %}
-                x-data="S3Upload('{% url "connector_s3:object_upload" datasource.id %}', '{{ sync_url }}?synchronous', '{{ object.key }}')"
-                x-init="init($el)"
-                x-swap="objects_section"
-                x-html="refreshedHtml"
-            {% endslot %}
-            {% slot actions %}
-                {% include "connector_s3/components/upload.html" with bucket=datasource %}
-            {% endslot %}
-            {% slot content %}
-                {{ datagrid }}
-            {% endslot %}
-        {% endembed %}
-    {% endif%}
+    {% static "connector_s3/img/symbol.svg" as symbol %}
```

Comment:
The tab bar for a file seems useless: it contains only one tab. Maybe it could be removed?

Futhermore, if object.type != directory => don't show tabs, maybe this template should be different for file or directory?
File: download button + {{ object_card }}
Directory: tabs + {{ object_grid }} + {{ object_card }}
Maybe too early generalisation?

Reply:
I agree but this can be done afterward. I'm also hesitating because we should put back the metadata / comments as soon as we have a solution for preserving metadata when moving or renaming files.